packages/website/src/theme/NotFound/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useLocation } from '@docusaurus/router';
+import { PageMetadata } from '@docusaurus/theme-common';
+import Layout from '@theme/Layout';
+import React from 'react';
+
+import styles from './styles.module.css';
+
+export default function NotFound(): React.JSX.Element {
```

Comment:
```suggestion
export default function NotFound(): JSX.Element {
```

Reply:
`React.` is actually the new way of doing things - the global `JSX` type is deprecated.

Reply:
Ah, didn't know that. This is so much longer though :(